docs/UserGuide.md

```diff
@@ -11,117 +12,273 @@
+
+Format: `find [-e|-t] KEYWORD [MORE_KEYWORDS]`
+
+> `KEYWORDS` are case sensitive. Events/Tasks which contain at least one keyword in their names will be returned. 
+
+Examples:
+* `find CS2103`
+  Shows tasks and events which have CS2103 (ignoring cases) in their names or description.
+* `find -e CS2103`
+  Returns relevant information of "CS2103 Exam" but not "cs2103 Exam"
+* `find -t CS2106 CS2103`
+  Returns any tasks or events having "CS2106", "CS2103" in their names. 
+
+
+#### Clearing of tasks
+Clears all completed tasks or clears all tasks. <br>
+
+Format: `clear -t [-a]`
+
+> An `-a` optional flag will request the TaskBook to clear all tasks, both marked done and not yet marked done. 
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+* `clear -t` <br>
+  Clears tasks that are marked done.
+* `clear -t -a` <br>
+  Clears all tasks.
+
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Clearing of events
+Clears all completed events or clears all events. <br>
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Format: `clear -e [-a]`
+
+> An `-a` optional flag will request the TaskBook to clear all events. 
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `clear -e` <br>
+  Clears events that are marked done.
+* `clear -e -a` <br>
+  Clears all events.
+
+  
+#### Clearing of tasks and events
+Clears all completed tasks and events or clears all tasks and events. <br>
 
-#### Clearing all entries : `clear`
-Clears all entries from the address book.<br>
-Format: `clear`  
+Format: `clear [-a]`
+
+> An `-a` optional flag will request the TaskBook to clear all tasks and events.
+
+Examples: 
+* `clear` <br>
+  Clears all completed tasks and events.
+* `clear -a` <br>
+  Clears all tasks and events.
+  
+  
+#### Exiting the program
+Format : `exit`
 
-#### Exiting the program : `exit`
-Exits the program.<br>
-Format: `exit`  
 
-#### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with 
-       the file that contains the data of your preious Address Book.
+       the file that contains the data of your previous Task Book.
        
```

Comment:
In future, you might want to use partial screenshots to show GUI changes for some commands... For example, if user types in a wrong command, it will show a red box, and a feedback message. :)